userguide/logout.rst

```diff
@@ -4,7 +4,11 @@
 Log Out
 =======
 
-Click the :guilabel:`Log Out` entry in the %brand% GUI to log out.
+Click :guilabel:`Settings` in the upper right corner then
+:guilabel:`Log Out` in the %brand% GUI to log out.
 
-After logging out, a message appears with a link to log back in. When
-logging back in, the *root* password is required.
+A popup window confirms the choice. Click :guilabel:`Ok` and the
```

Comment:
This all seems very wordy and roundabout.  "The system continues the log out process." is not very clear.
How about "Confirm the choice to log out."?